__tests__/tilt/tokenbridge.test.ts

```diff
@@ -65,7 +65,7 @@ describe("Tilt Token Bridge Tests", () => {
             const [msgid] = await eth.parseTransaction(txns[txns.length - 1].txid)
             expect(isWormholeMessageId(msgid)).toBeTruthy()
 
-            const vaa = await wh.getVAA(msgid.chain, msgid.emitter, msgid.sequence, "AttestMeta")
+            const vaa = await wh.getVAA(msgid.chain, msgid.emitter, msgid.sequence, "TokenBridge-AttestMeta")
```

Comment:
Can we use a diff char to delimit namespace-payload type? maybe `:` or `/` or `.`?  

I don't have a good reason besides a hyphen feeling wrong 😆